rake_helpers/generate_final_csvs.rb

```diff
@@ -78,4 +78,47 @@
     wikidata_parties.last.each { |p| warn "  Missing: #{p[:name]} (#{p[:id]})" } if wikidata_parties.first.count > 0 && wikidata_parties.last.count <= 5
   end
 
+  desc 'Build the Positions file'
+  task :positions => ['sources/wikidata/positions.json', 'ep-popolo-v1.0.json'] do
+
+    filter_file   = 'sources/manual/position-filter.json'
+    position_file = "unstable/positions.csv"
+    warn "Creating #{position_file}"
+
+    positions = JSON.parse(File.read('sources/wikidata/positions.json'), symbolize_names: true) 
+    filter    = File.exist?(filter_file) ? JSON.parse(File.read(filter_file), symbolize_names: true).each do |s, fs|
+      fs.each { |f| f.delete :count }
+    end : { exclude: [], include: [] }
```

Comment:
I think these lines might be clearer as an if statement, the ternary operator is a bit hidden otherwise!

```ruby
filter = if File.exist?(filter_file)
  JSON.parse(File.read(filter_file), symbolize_names: true).each do |s, fs|
    fs.each { |f| f.delete :count }
  end
else
  { exclude: [], include: [] }
end
```